src/PIL/features.py

```diff
@@ -119,9 +120,9 @@ def get_supported_codecs() -> list[str]:
 
 
 features = {
```

Comment:
```suggestion
features: dict[str, tuple[str, str | bool, str | None]] = {
```
This should fix the type hint error.

Reply:
Thanks! This helps